nebula/src/nebula/video_storage/storage.py

```
@@ -0,0 +1,297 @@
+        video_dir.mkdir(parents=True, exist_ok=True)
+
+        # Save original video file
+        video_path = video_dir / filename
+        with open(video_path, "wb") as f:
+            f.write(video_data)
+
+        # Convert to HLS format
+        try:
+            duration = self._convert_to_hls(video_dir, video_path)
+        except Exception as e:
+            logger.error("Error converting video to HLS: %s", e)
```

Ruff (0.14.3)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)